ampersand-form-view.js

```diff
@@ -1,54 +1,65 @@
 /*$AMPERSAND_VERSION*/
-var Events = require('ampersand-events');
+var View = require('ampersand-view');
 var isFunction = require('lodash.isfunction');
-var assign = require('lodash.assign');
 var result = require('lodash.result');
-var classExtend = require('ampersand-class-extend');
-
-
-function FormView(opts) {
-    opts = opts || {};
-    this.el = opts.el;
-    this.validCallback = opts.validCallback || this.validCallback || function () {};
-    this.submitCallback = opts.submitCallback || this.submitCallback || function () {};
-    this.clean = opts.clean || this.clean || function (res) { return res; };
-
-    if (opts.data) this.data = opts.data;
-    if (opts.model) this.model = opts.model;
+var noop = function(){};
+
+module.exports = View.extend({
+
+    derived: {
+        data: {
+            fn: function () {
+                var res = {};
+                for (var key in this._fieldViews) {
+                    if (this._fieldViews.hasOwnProperty(key)) {
+                        res[key] = this._fieldViews[key].value;
+                    }
+                }
+                return this.clean(res);
+            }
+        },
+        cache: false
```

Comment:
`cache: false` should be inside the `data` object - moving it up one line would suffice! ;-)